Jenkinsfile

```diff
@@ -81,10 +81,12 @@ def pcStage(String stageName, Closure body) {
 
     checkout scm
 
-    def dockerArgs = '--user=root -v /tmp/comma_download_cache:/tmp/comma_download_cache -v /tmp/scons_cache:/tmp/scons_cache';
+    def dockerArgs = '--user=batman -v /tmp/comma_download_cache:/tmp/comma_download_cache -v /tmp/scons_cache:/tmp/scons_cache';
     docker.build("openpilot-base:build-${env.GIT_COMMIT}", "-f Dockerfile.openpilot_base .").inside(dockerArgs) {
       timeout(time: 20, unit: 'MINUTES') {
         try {
+          sh "sudo chown -R batman:batman /tmp/scons_cache"
```

Comment:
similar to the devcontainer, this can be removed eventually when there are no jenkins jobs running as root

Reply:
make sure to leave a comment for stuff like this